app_allocator/classes/entity.py

```diff
@@ -43,15 +43,6 @@ def add_property(self, property, data=None):
         if value is not None:
             self.properties[property.name] = value
 
-    def add_fields_to_name(self, fields):
```

Comment:
This was useful for debugging, but I see why you removed it.  We should discuss debugging at some point.